spec/browserstack_helper.rb

```diff
@@ -1,40 +1,40 @@
-# -*- coding: utf-8 -*-
```

Comment:
Ah, right. Yes. That file is also poorly named: it's not just for BrowserStack, but also sets up the environment when running features locally.